js/main.js

```diff
@@ -10,6 +10,13 @@ Site = {
       _this.onResize();
     });
 
+    $(window).bind('resizeEnd', function() {
+      if ($('.journal-container').length) {
```

Comment:
this should call a `Site.Journal.Layout()` function or something named like that